frontend/device/input.lua

```diff
@@ -372,15 +373,16 @@ function Input:setTimeout(slot, ges, cb, origin, delay)
     if input.setTimer then
         -- If GestureDetector's clock source probing was inconclusive, do this on the UI timescale instead.
         if clock_id == -1 then
-            deadline = TimeVal:now() + delay
+            deadline = time.now() + delay
             clock_id = C.CLOCK_MONOTONIC
         else
             deadline = origin + delay
         end
         -- What this does is essentially to ask the kernel to wake us up when the timer expires,
         -- instead of ensuring that ourselves via a polling timeout.
         -- This ensures perfect accuracy, and allows it to be computed in the event's own timescale.
-        timerfd = input.setTimer(clock_id, deadline.sec, deadline.usec)
+        local sec, usec = time.split_s_us(deadline)
```

Comment:
`to_s_us(deadline)` for consistency ? Although `split_s_us()` is quite clear.

Reply:
We use this only in `input.lua`. So I wouldn't change that. This is a hint.

Reply:
I'd probably have preferred to have `tuple` somewhere in there, but, I can't quite think of something that works while having the units as a suffix and not a prefix :/.